cockroachcloud/authorization.md

```diff
@@ -63,36 +52,86 @@ Org Administrators can manage the organization and its members, clusters, and co
+This role will is deprecated in favor of more fine-grained roles introduced below.
 {{site.data.alerts.end}}
 
-### Cluster Developer
+### Organization member
+
+This default role is granted to all organization users once they are invited. It grants no permissions to perform cluster or org actions.
+
+### Org Administrator
+
+Users with this role on an organization can:
+
+- [invite users to join that organization](managing-access.html#invite-team-members-to-an-organization)
+- [create service accounts](managing-access.html#create-a-service-account)
+- grant and revoke roles for both [users](managing-access.html#manage-an-organizations-users) and [service accounts](managing-access.html#manage-service-accounts).
+
+This role replaces the now deprecated [Org Administrator (legacy)](#org-administrator-legacy).
 
-Cluster Developers can view the details of clusters and can change their IP allowlist configuration. This role can be granted for specific clusters or for all clusters in the organization.
+### Billing Coordinator
+
+Users with this role in an organization can [manage billing for that organization](billing-management.html) through the {{ site.data.products.db }} console billing page at `https://cockroachlabs.cloud/billing/overview`.
+
+Note that billing can also be managed by the [Org Administrator (legacy) role](#org-administrator-legacy).
+
+### Usage Reader
+
+This is a read-only role that allows a user to access usage metrics for an organization.
+
+### Cluster Operator
+
+This role grants different permissions to users and service accounts.
+
+Users with this role can perform the following console actions:
+
+- Configure authorize network connections for dedicated private clusters ([allowed IP ranges](network-authorization.html#ip-allowlisting), [GCP VPC Peering](network-authorization.html#vpc-peering), or [AWS PrivateLink](network-authorization.html#aws-privatelink))
+- Configure [egress perimeter controls](egress-perimeter-controls.html)
+
+Service accounts with this role can perform the following API functions:
+
+- [Manage Customer-Managed Encryption Keys (CMEK) for Dedicated Clusters](managing-cmek.html)
+- [Export Logs From a CockroachDB Dedicated Cluster
+](export-logs.html)
+- [Export Metrics From a CockroachDB Dedicated Cluster](export-metrics.html)
+
+This role can be granted for one or more specific clusters, or for all clusters in the organization.
 
 ### Cluster Administrator
 
-Cluster Administrators can manage SQL users and roles for a cluster, and perform all management functions on that cluster (like enabling CMEK, configuring Log Export). This role can be granted for one or more specific clusters, or for all clusters in the organization.
+Cluster Administrators can [provision SQL users for a cluster using the console](managing-access.html#create-a-sql-user), as well as perform all of the [Cluster Operator actions](#cluster-operator).
```

Comment:
In addition to SQL users, they can also (1) grant access to cloud users and service accounts, and (2) create new service accounts.

Reply:
They can create service accounts, yes. (The team had a lot of conversations about that one!) For org roles, they can't manage everything, but they do have permission to grant the Cluster {Admin, Operator, Developer} roles on clusters they administer.